lib/make-logger.js

```diff
@@ -2,9 +2,15 @@
 
 function noop() {}
 
-function makeLoggerForSeverity(severity, stream) {
+function makeLoggerForSeverity(severity, stream, isMetric) {
     return function(message, jsonPayload) {
-        stream.write({ ...jsonPayload, severity, message, eventTime: new Date().toISOString() });
+        stream.write({
+            ...jsonPayload,
+            isMetric,
```

Comment:
Perhaps this could be more generic... It could unpack some additional data, like:
```javascript
stream.write({
   ...jsonPayload,
   ...additional,
   severity,
   message,
   eventTime: new Date.toISOString()
});
```
By doing that, the log levels become something like:
```javascript
[
    { severity: 'debug', stream: out },
    { severity: 'info', stream: out },
    { severity: 'metric', stream: out, additional: { isMetric: true } },
    { severity: 'warning', stream: out },
    { severity: 'error', stream: out },
    { severity: 'alert', stream: out }
]
```
It _might_ be over-engineering it though...

Reply:
I like that. Makes things official.

Reply:
I can see this evolving into better handling of child loggers in the future. I'll ponder...